diagnostic.md

```diff
@@ -3,67 +3,72 @@
 Using `inspect element`, write four selectors found on the main page for this
 GitHub repo.
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+element.style
+div
+body
+html
```

Comment:
Good, `element.style` is more of a feature of the google chrome developer tools, its basically a way to play around with the current element you have highlighted without actually changing your style sheet.